crates/libs/windows/src/core/mod.rs

```diff
@@ -113,3 +113,13 @@ pub fn as_mut_ptr_or_null<T>(value: &mut [T]) -> *mut T {
         value.as_mut_ptr()
     }
 }
+
+fn wide_trim_end(mut wide: &[u16]) -> &[u16] {
```

Comment:
Nit: it would be nice to provide a short doc about why this helper is necessary. It's obvious in this PR, but in the future, contributors may wonder why we need this.